sydent/http/servlets/emailservlet.py

```diff
@@ -122,18 +129,19 @@ def render_GET(self, request):
 
         request.setHeader("Content-Type", "text/html")
         res = open(templateFile).read() % {"message": msg}
+
         return res.encode("UTF-8")
 
     @jsonwrap
-    def render_POST(self, request):
+    def render_POST(self, request: Request) -> JsonDict:
         send_cors(request)
 
         if self.require_auth:
             authV2(self.sydent, request)
 
         return self.do_validate_request(request)
 
-    def do_validate_request(self, request):
+    def do_validate_request(self, request: Request) -> Dict[str, Union[bool, str]]:
```

Comment:
I think this is returning a `JsonDict` since it is directly used as the result of `render_POST`?

```suggestion
    def do_validate_request(self, request: Request) -> JsonDict:
```